README.md

```diff
@@ -136,13 +139,15 @@ export ODBCINI=$II_SYSTEM/ingres/files/odbc.ini
 ```python
 import sys
 import sqlalchemy
+#import ingres_sa_dialect
 
 print('Python %s on %s' % (sys.version, sys.platform))
 print('SQLAlchemy %r' % sqlalchemy.__version__)
 con_str = 'ingres:///demodb'  # local demodb
 #con_str = 'ingres://dbuser:PASSWORD@HOSTNAME:27832/db'  # remote database called "db"
 print(con_str)
-print(ingres_sa_dialect.base.dialect().create_connect_args(url=sqlalchemy.engine.make_url(con_str)))
+# If the next line is uncommented, need to also uncomment: import ingres_sa_dialect
+#print(ingres_sa_dialect.base.dialect().create_connect_args(url=sqlalchemy.engine.make_url(con_str)))
```

Comment:
alternative is to import and print on the same line with a semi-colon which would make this cleaner (whilst still being dirty ;-))